spyder/plugins/toolbar/container.py

```diff
@@ -0,0 +1,322 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Toolbar Container.
+"""
+
+# Standard library imports
+from collections import OrderedDict
+
+# Third party imports
+from qtpy.QtCore import QSize, Qt, Signal, Slot
+from qtpy.QtWidgets import QMenu, QToolBar
+
+# Local imports
+from spyder.api.exceptions import SpyderAPIError
+from spyder.api.translations import get_translation
+from spyder.api.widgets import PluginMainContainer
+from spyder.api.widgets.toolbars import ApplicationToolBar
+from spyder.plugins.toolbar.api import ApplicationToolBars
+
+# Localization
+_ = get_translation('spyder')
+
+
+# --- Constants
+# ------------------------------------------------------------------------
+
+class ToolBarMenus:
+    ToolBarsMenu = "toolbars_menu"
+
+
+class ToolBarsMenuSections:
+    Main = "main_section"
+    Secondary = "secondary_section"
+
+
+class ToolBarActions:
+    ShowToolBars = "show_toolbars_actions"
+
+
+class ToolBarContainer(PluginMainContainer):
+    DEFAULT_OPTIONS = {
+        'last_visible_toolbars': [],
+        'toolbars_visible': True,
+    }
+
+    def __init__(self, name, plugin, parent=None, options=DEFAULT_OPTIONS):
+        super().__init__(name, plugin, parent=parent, options=options)
+
+        self._APPLICATION_TOOLBARS = OrderedDict()
+        self._ADDED_TOOLBARS = OrderedDict()
+        self._toolbarslist = []
+        self._visible_toolbars = []
+
+    # --- Private Methods
+    # ------------------------------------------------------------------------
+    def _save_visible_toolbars(self):
+        """Saves the name of the visible toolbars in the options."""

```

Comment:
Docstring should start in infinitive: Saves -> Save

Reply:
Fixed.